VRCFaceTracking/Params/Expressions/Legacy/Lip/UnifiedSRanMapper.cs

```diff
@@ -0,0 +1,66 @@
+
+            { SRanipal_LipShape_v2.MouthPout, exp => (exp.Shapes[(int)UnifiedExpressions.LipPuckerLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipPuckerRight].AdjustedWeight) / 2.0f },
+
+            { SRanipal_LipShape_v2.MouthSmileRight, exp => exp.Shapes[(int)UnifiedExpressions.MouthSmileRight].AdjustedWeight },
+            { SRanipal_LipShape_v2.MouthSmileLeft, exp => exp.Shapes[(int)UnifiedExpressions.MouthSmileLeft].AdjustedWeight },
+            { SRanipal_LipShape_v2.MouthSadRight, exp => Math.Min(1, exp.Shapes[(int)UnifiedExpressions.MouthFrownRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.MouthStretchRight].AdjustedWeight) },
+            { SRanipal_LipShape_v2.MouthSadLeft, exp => Math.Min(1, exp.Shapes[(int)UnifiedExpressions.MouthFrownLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.MouthStretchLeft].AdjustedWeight) },
+
+            { SRanipal_LipShape_v2.CheekPuffLeft, exp => exp.Shapes[(int)UnifiedExpressions.CheekPuffLeft].AdjustedWeight },
+            { SRanipal_LipShape_v2.CheekPuffRight, exp => exp.Shapes[(int)UnifiedExpressions.CheekPuffRight].AdjustedWeight },
+            { SRanipal_LipShape_v2.CheekSuck, exp => (exp.Shapes[(int)UnifiedExpressions.CheekSuckLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.CheekSuckRight].AdjustedWeight) / 2.0f },
+
+            { SRanipal_LipShape_v2.MouthUpperUpRight, exp => exp.Shapes[(int)UnifiedExpressions.MouthUpperUpRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipFunnelUpperRight].AdjustedWeight },
+            { SRanipal_LipShape_v2.MouthUpperUpLeft, exp => exp.Shapes[(int)UnifiedExpressions.MouthUpperUpLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipFunnelUpperLeft].AdjustedWeight },
+            { SRanipal_LipShape_v2.MouthLowerDownRight, exp => exp.Shapes[(int)UnifiedExpressions.MouthLowerDownRight].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipFunnelLowerRight].AdjustedWeight },
+            { SRanipal_LipShape_v2.MouthLowerDownLeft, exp => exp.Shapes[(int)UnifiedExpressions.MouthLowerDownLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipFunnelLowerLeft].AdjustedWeight },
+
+            { SRanipal_LipShape_v2.MouthUpperInside, exp => (exp.Shapes[(int)UnifiedExpressions.LipSuckUpperLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipSuckUpperRight].AdjustedWeight) / 2.0f },
+            { SRanipal_LipShape_v2.MouthLowerInside, exp => (exp.Shapes[(int)UnifiedExpressions.LipSuckLowerLeft].AdjustedWeight + exp.Shapes[(int)UnifiedExpressions.LipSuckLowerRight].AdjustedWeight) / 2.0f },
+
+            { SRanipal_LipShape_v2.MouthLowerOverlay, exp => exp.Shapes[(int)UnifiedExpressions.MouthRaiserLower].AdjustedWeight},
+
+            { SRanipal_LipShape_v2.TongueLongStep1, exp => exp.Shapes[(int)UnifiedExpressions.TongueOut].AdjustedWeight},
+            { SRanipal_LipShape_v2.TongueLongStep2, exp => exp.Shapes[(int)UnifiedExpressions.TongueOut].AdjustedWeight},
```

Comment:
TongueOut would be TongueLongStep1 and TongueLongStep2 both 1. So TongueLongStep1 should be half of Tongue Out and TongueLongStep2 should also be half of TongueOut

Reply:
That's right, let me make that change. It should still work consistently when using TongueOut from the SRanipal combined parameters. I'll make it so that TongueStep1 uses 0-0.5 range and Step2 uses 0.5-1 range from Unified TongueOut